docs/server/configuration.md

```diff
@@ -26,23 +26,56 @@ Temporal Server configuration is found in `development.yaml` and may contain the
 The `global` section contains process-wide configuration. See below for a minimal configuration (optional parameters are commented out.)
 
 ```yaml
-#global:
-#membership:
-#maxJoinDuration: 30s
-#broadcastAddress: "127.0.0.1"
-#pprof:
-#port: 7936
-#tls:
-#... <see below>
+global:
+  membership:
+    broadcastAddress: "127.0.0.1"
+  metrics:
+    prometheus:
+      framework: "opentelemetry"
+      listenAddress: "127.0.0.1:8000"
 ```
 
 ### membership
 
 The `membership` section controls the following membership layer parameters:
 
-- `maxJoinDuration` - The amount of time the service will attempt to join the gossip layer before failing.
-- `broadcastAddress` - Used as the address that is communicated to remote nodes to connect on.
-  - This is generally used when BindOnIP would be the same across several nodes (ie: 0.0.0.0) and for nat traversal scenarios. `net.ParseIP` controls the supported syntax. Note: Only IPV4 is supported.
+- `maxJoinDuration` - The amount of time the service will attempt to join the gossip layer before failing. Default is 10s.
+- `broadcastAddress` - Used by gossip protocal to communicate with other hosts in the same cluster for membership info.
+  - Use IP address that is reachable by other hosts in the same cluster. If there is only one host in the cluster, you can use 127.0.0.1. Check `net.ParseIP` for supported syntax, only IPv4 is supported.
+
+### metrics
+
+`metrics` - contains config for metrics subsystem.
+
+- `prefix` - the prefix to all outgoing metrics.
+- `tags` - the set of key-value pairs to be reported as part of every metric.
+- `excludeTags` - a map from tag name string to tag values string list. This is useful to exclude some tags that might have unbounded cardinality. The value string list can be used to whitelist values of that excluded tag to continue to be included. For example, if you want to exlude task_queue because it has unbounded cardinality, but you still want to see a whitelisted value for task_queue.
```

Comment:
```suggestion
- `prefix`: The prefix to all outgoing metrics.
- `tags`: The set of key-value pairs to be reported as part of every metric.
- `excludeTags`: A map from tag name string to tag values string list.
This is useful to exclude some tags that might have unbounded cardinality.
The value string list can be used to whitelist values of that excluded tag to continue to be included.
For example, if you want to exclude `task_queue` because it has unbounded cardinality, but you still want to see a whitelisted value for `task_queue`.
```